src/commands/createNewProject/PythonProjectCreator.ts

```diff
@@ -54,8 +53,14 @@ export class PythonProjectCreator extends ScriptProjectCreatorBase {
         if (!await validateFuncCoreToolsInstalled(true /* forcePrompt */, funcCoreRequired)) {
             throw new UserCancelledError();
         }
-        await this.validatePythonVersion();
-        await this.createVirtualEnviornment();
+        if (await fse.pathExists(path.join(this.functionAppPath, funcEnvName))) {
+            const input: MessageItem = await ext.ui.showWarningMessage(localize('funcEnvExists', 'Python Virtual Environment already exists.  Overwrite?', funcEnvName), { modal: true }, DialogResponses.yes, DialogResponses.no, DialogResponses.cancel);
+            if (input === DialogResponses.yes) {
+                await createVirtualEnviornment(this.functionAppPath);
```

Comment:
nit: Personally I prefer to add a Boolean rather than duplicating function calls even when small like this.